apps/vr-tests/src/stories/Button.stories.tsx

```diff
@@ -166,10 +166,13 @@ storiesOf('Button Split', module)
   .addStory('Root', () => <DefaultButton {...commandProps} split={true} />, { rtl: true })
   .addStory('Disabled', () => <DefaultButton {...commandProps} disabled={true} split={true} />)
   .addStory('Checked', () => <DefaultButton {...commandProps} checked={true} split={true} />)
-  .addStory('Primary', () => <DefaultButton {...commandProps} primary={true} split={true} />)
-  .addStory('Primary Disabled', () => (
+  .addStory('Primary Theme', () => <DefaultButton {...commandProps} primary={true} split={true} />)
+  .addStory('Primary Theme with Disabled State', () => (
```

Comment:
no changes needed for these 2 LoC (169 and 170). `Primary` in here is not the theme, it's only the type of the button that happens to use `themePrimary` palette slot for color. Revert this changes please.

Reply:
This way you avoid to much noise in the screener tests.

Reply:
Thanks @Vitalius1 !